protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -17,22 +77,47 @@ use secp256k1::{ellswift::ElligatorSwift, Keypair, Secp256k1, SecretKey};
 
 const VERSION: u16 = 0;
 
+/// Represents the state and operations of the Noise NX protocol responder.
+///
+/// Manages the state of the responder during the Noise NX protocol handshake process. It holds the
+/// necessary cryptographic keys, handshake state, and intermediate values needed to perform the
+/// key exchange and establish a secure connection with the initiator.
 pub struct Responder {
+    /// Cipher used for encrypting and decrypting messages during the handshake.
+    ///
+    /// It is initialized once enough information is available from the handshake process.
     handshake_cipher: Option<ChaCha20Poly1305>,
+    /// Optional static key used in the handshake. This key may be derived from the pre-shared key
+    /// (PSK) or generated during the handshake.
     k: Option<[u8; 32]>,
```

Comment:
here and in the `initiator` (very) short names are used for the variables. wdyt about using full names here and else-where, even if its too long? that should be a task for the refactor phase though if we agree on that.

Reply:
We are going by the noise protocol naming convention.